project/config/lgbcni/config/workflows.workflow.nics_editorial_workflow.yml

```diff
@@ -10,7 +10,6 @@ dependencies:
     - node.type.feature
     - node.type.featured_content_list
     - node.type.news
-    - node.type.publication
```

Comment:
Because you have changed the publication content type machine name (D9) you have lost your workflow moderation.

Reply:
Thanks this has now been updated